js/index.js

```diff
@@ -20,4 +20,58 @@ for(let i = 0; i < skills.length; i++){
 	let li = document.createElement('li');
 	li.innerHTML = skills[i];
 	skillsList.appendChild(li);
-}
\ No newline at end of file
+}
+
+
+let messageForm = document.querySelector('[name="leave_message"]');
+messageForm.addEventListener("submit", (e) => {
+	
+	
+	//can't use the function declaration way!
+	//MUST use arrow syntax.
+	
+	e.preventDefault();
+	
+
+
+	
+	let name = "myName";
+	let email = "myEmail";
+	let message = "myMessage";
+
+	
+	name = e.target.name.value;
+	email = e.target.email.value;
+	message = e.target.message.value;
```

Comment:
This works, but you could also directly initialize and assign the variables to the values, i.e. `const name = e.target.name.value`.